x-pack/plugins/apm/public/components/app/service_inventory/index.tsx

```diff
@@ -265,6 +266,34 @@ export function ServiceInventory() {
     [tiebreakerField]
   );
 
+  const setApplicationContext =
+    useApmPluginContext().observabilityAIAssistant.service
+      .setApplicationContext;
+
+  useEffect(() => {
+    if (mainStatisticsStatus === FETCH_STATUS.FAILURE) {
+      return setApplicationContext({
+        description: 'The services have failed to load',
+      });
+    }
+
+    if (mainStatisticsStatus === FETCH_STATUS.LOADING) {
```

Comment:
`import {  isPending } from '../../../../hooks/use_fetcher';` 
```suggestion
    if (isPending(mainStatisticsStatus)) {
```